gems/pending/appliance_console/database_configuration.rb

```diff
@@ -79,7 +79,6 @@ def activate
     end
 
     def post_activation
-      ServiceGroup.new.restart_services
       LinuxAdmin::Service.new("evmserverd").enable.start
```

Comment:
Yup, they all get enabled in the kickstart post section [here](https://github.com/ManageIQ/manageiq-appliance-build/blob/master/kickstarts/partials/post/systemd.ks.erb) so they'll be running at this point.